api/test/nostd/shared_ptr_test.cc

```diff
@@ -179,3 +180,26 @@ TEST(SharedPtrTest, Sort)
 
   EXPECT_EQ(nums, nums2);
 }
+
+TEST(SharedPtrTest, Sort)
+{
+  SharedPtrTest_Sort();
+}
+
+static void SharedPtrTestSort(benchmark::State &state)
+{
+  for (auto _ : state)
+  {
+    SharedPtrTest_Sort(10000);
+  }
+}
+BENCHMARK(SharedPtrTestSort);
```

Comment:
Can you move this into its own benchamrk CC file?


In bazel there is a "smoketest" for benchmarks which will run them as a test, but if we can keep them separated via the benchmark bazel rule, we'll be able to automatically generate performance reports on every CL going forward (something our team is working on).

Reply:
Sure. Will do.